_inc/client/components/jetpack-notices/notice-action-disconnect.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import NoticeAction from 'components/notice/notice-action';
+import { disconnectSite } from 'state/connection';
+
+class NoticeActionDisconnect extends React.Component {
+	static propTypes = {
+		icon: PropTypes.string,
+	};
+
+	handleDisconnectClick = () => {
```

Comment:
I think it would be nice to send a tracks event here, so we can determine if the user disconnected from this notice. 

An event name like `jetpack_termination_error_notice_click` should work